include/openthread/border_agent.h

```diff
@@ -546,6 +546,24 @@ void otBorderAgentEphemeralKeySetCallback(otInstance                       *aIns
  */
 const char *otBorderAgentEphemeralKeyStateToString(otBorderAgentEphemeralKeyState aState);
 
+/**
+ * Generates a Thread Administration One-Time Passcode (TAP) and starts ephemeral key with it.
+ *
+ * This function generates an 8-digit random number, appends a Verhoeff checksum
+ * as the 9th digit to create a 9-digit TAP, and automatically starts the ephemeral
+ * key with the generated passcode.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ * @param[out] aTap       A pointer to buffer to store the generated TAP string (must be at least 10 bytes).
```

Comment:
In this same file we can also add a new constant around line 388-408, as follows:

```
/**
 * Fixed length of the Thread Administration Sharing One-Time Passcode (TAP) string.
 * This includes the 9-digit passcode and a null termination character.
 */
#define OT_BORDER_AGENT_EPHEMERAL_TAP_STR_LENGTH (10)
```

Then we can refer to length of the buffer needing to be >=  `OT_BORDER_AGENT_EPHEMERAL_TAP_STR_LENGTH` and also use this constant in other places in the code where `10` is now included.